aspnetcore/web-api/jsonpatch/samples/3.1/mvc/Models/ProductCategoryConverter.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+
+namespace JsonPatchSample.Models
+{
+    public class ProductCategoryConverter : JsonConverter
+    {
+        public override bool CanConvert(Type objectType)
+        {
+            return true;
+        }
+
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        {
+            // Load has to happen even if we are not using it. If removed, deserialization will throw an error -
+            //"Additional text found in JSON string after finishing deserializing object."
```

Comment:
?

Reply:
You are right, based on ASP.NET Core don't need mvc and api both samples. I deleted mvc sample.

Reply:
You are right, based on ASP.NET Core don't need mvc and api both samples. I deleted mvc sample.